CHANGELOG.md

```diff
@@ -0,0 +1,64 @@
+
+# Version 2.0.0
+
+## Breaking changes
+
+* support for Python 3.5 is dropped
+
+* `import versions` is replaced by:
+
+```python
+import vindaloo
+versions = vindaloo.app.versions
+```
+
+* attributes (`containers`, `ports` ...) on `Deployment`, `Service`, `CronJob`, 
+  e.g. are moved to full path corresponding to structure of k8s manifest:
+
+```python
+-JOB.containers['foo']['command']
++JOB.spec.template.spec.containers['foo']['command']
+-SERVICE.load_balancer_ip
++SERVICE.spec.loadBalancerIP
+-SERVICE.ports['http']['nodePort'
++SERVICE.spec.ports['http']['nodePort']
+```
+
+* init parameter `annotations` of `Deployment`, `CronJob` and `Job` now sets
+`metadata.annotations` in manifest.
+  
+* init parameter `spec_annotations` to `Deployment`, `CronJob` and `Job` is added
+which sets:
+  * `spec.template.metadata.annotations` in `Deployment`
+  * `spec.jobTemplate.spec.template.metadata.annotations` in `CronJob`
+  
+## Improvements
+
+* Any parameter in manifest spec can now be set, user is not limited to attributes on `Deployment` e.g. 
+
+```python
+DEPLOYMENT.spec.something.foo.bar = 'boo'
+```
+
+* Objects can be created without init params:
+
+```python
+SERVICE = Service()
```

Comment:
Premyslim, zebychom mohli pristupovat k tem objektum jako "dict", tzn. vc. kopirovaciho konstruktoru.
```python
>>> x = {"a": 1}
>>> dict(x)
{'a': 1}
```
a v kontstuktorech popovat z kwargu parametry.


Ale z toho co se divam, tak uz asi to umime, ne?
```python
List({'foo': JOB_CONTAINER})
```

Reply:
JJ, umime :)